app-service-proxy/internal/server/transport.go

```diff
@@ -72,6 +72,24 @@ func (t *RewritingTransport) RoundTrip(req *http.Request) (*http.Response, error
 		return nil, err
 	}
 
+	// Patch CSP header
+	cspHeaders := resp.Header.Values("Content-Security-Policy")
+	if len(cspHeaders) > 0 {
+		var newCSPs []string
+		for _, csp := range cspHeaders {
+			if strings.Contains(csp, "frame-ancestors 'none'") {
+				csp = strings.ReplaceAll(csp, "frame-ancestors 'none'", "frame-ancestors 'self'; patched-by-asp")
```

Comment:
I think you should leave out `patched-by-asp` - it's best to just stick to standard values.